nixos/modules/programs/command-not-found/command-not-found.nix

```diff
@@ -8,64 +8,87 @@
 with lib;
 
 let
-
+  cfg = config.programs.command-not-found;
   commandNotFound = pkgs.substituteAll {
     name = "command-not-found";
     dir = "bin";
     src = ./command-not-found.pl;
     isExecutable = true;
     inherit (pkgs) perl;
+    inherit (cfg) dbPath;
     perlFlags = concatStrings (map (path: "-I ${path}/lib/perl5/site_perl ")
       [ pkgs.perlPackages.DBI pkgs.perlPackages.DBDSQLite pkgs.perlPackages.StringShellQuote ]);
   };
 
 in
 
 {
+  options.programs.command-not-found = {
+
+    enable = mkOption {
+      default = true;
```

Comment:
I would even argue to disable this at the moment, because the sqlite database is almost empty since 16.09:

16.03:
```
$ sqlite3 /nix/var/nix/profiles/per-user/joerg/channels/nixos/programs.sqlite "select count(*) from Programs;"
23209
```

16.09:
```
$ sqlite3 /nix/var/nix/profiles/per-user/joerg/channels/nixos/programs.sqlite "select count(*) from Programs;"
653
```

17.03:
```
$ sqlite3 /nix/var/nix/profiles/per-user/joerg/channels/nixos/programs.sqlite "select count(*) from Programs;"
343
```